### PR TITLE
Disabled invalid option in the form.  Story 9717

### PR DESCRIPTION
**Pull request by val99erie**
The controller error was already fixed as part of story 9713.  I just made a change to the form to disable the radio button option for the current version so that the user cannot submit an invalid version.

### CODE REVIEW

File: app/views/generic_files/_versioning.html.erb [hunk 1]
```diff
@@ -1,33 +1,38 @@
 <div id="versioning_display" class="tab-pane">
   <h2>Versions</h2>
   <div class="well">
+
     <%= form_for [sufia, @generic_file], html: { multipart: true } do |f| %>
       <%= hidden_field_tag('redirect_tab', 'versions') %>
-    <h3>Upload New Version</h3>
```
Comment (cam156) [inline app/views/generic_files/_versioning.html.erb]: Is there are reason you are swapping the order of the forms?

**Comment by jcoyne** [comment]
@val99erie Looks like you need to rebase so this can merge cleanly.

File: app/assets/stylesheets/sufia/_styles.scss [hunk 2]
```diff
@@ -82,6 +82,10 @@ label.required, span.required {
   }
 }
 
+label.disabled {
+  color: #666;
```
Comment (jcoyne) [inline app/assets/stylesheets/sufia/_styles.scss]: http://youtu.be/q9kT37eIkaY?t=1m49s

Comment (awead) [inline app/assets/stylesheets/sufia/_styles.scss]: :smiling_imp:

Comment (jcoyne) [inline app/assets/stylesheets/sufia/_styles.scss]: I wonder if `$input-bg-disabled` would be better than hard coding this.

Comment (jcoyne) [inline app/assets/stylesheets/sufia/_styles.scss]: Or `$gray-light`

**Comment by val99erie** [comment]
@jcoyne - I added a new gray color.  The existing $gray-medium was too light to see against the gray background, and the $gray-dark was indistinguishable from the text for the non-disabled options.

File: app/views/generic_files/_versioning.html.erb [hunk 3]
```diff
@@ -21,9 +21,11 @@
     <% @version_list.each do |version| %>
       <div class="form-group">
         <div class="col-xs-12">
-          <label class="radio">
-            <%= radio_button_tag(:revision, version.label, version.current?) %>
-            Restore from <%= version.created %> [by <%= version.committer %>]
+          <% disable_if_current_version = version.current? ? 'disabled' : '' %>
+          <label class="radio <%= disable_if_current_version %>">
```
Comment (cam156) [inline app/views/generic_files/_versioning.html.erb]: Not sure I like the layer of indirection here.  Why not just use version.current? ? 'disabled' : '' in the label.  disable_if_current_version is only being used here correct?

Comment (val99erie) [inline app/views/generic_files/_versioning.html.erb]: @cam156 You are correct.  I'll change it.

**Comment by jcoyne** [comment]
:+1:

**Pull request merged**